### PR TITLE
Remove Docker Compose: "version"

### PR DESCRIPTION
**Pull request by originalsouth**
### Changes
Remove Docker Compose: "version"
https://github.com/docker/compose/issues/11628

### Issue link
This is a trivial patch without an issue, making an issue would be superfluous be a superfluous burden for everybody involved.

### Demo
N/A

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.
- [x] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — COMMENTED** [review]
I've noticed this too a few days ago. Perhaps also apply this on the other Docker Compose files in the codebase too? (e.g. in the `.ci` folders)

**Comment by originalsouth** [comment]
> I've noticed this too a few days ago. Perhaps also apply this on the other Docker Compose files in the codebase too? (e.g. in the `.ci` folders)

Yeah, I was doing that, but than the whole CI breaks (so far; I don't want to add all the bogus experiments to the history so I decided to overwrite them) -- hence drafting [WIP], I will throw it in the group "when it's done" ;)

**Comment by originalsouth** [comment]
I guess we want to get rid of all the base files as they have no use anymore.

**Comment by originalsouth** [comment]
Do we want to keep [.ci/.ci/.env.test](https://github.com/minvws/nl-kat-coordination/blob/main/.ci/.env.test), it seems to be empty.

**Comment by originalsouth** [comment]
Ready for merge, removed all `*base.yml` and version statements.

This question remains, however (although not relevant to this patch per se):

> Do we want to keep [.ci/.ci/.env.test](https://github.com/minvws/nl-kat-coordination/blob/main/.ci/.env.test)? It seems to be empty.

**Comment by ammar92** [comment]
> Ready for merge, removed all `*base.yml` and version statements.
> 
> This question remains, however (although not relevant to this patch per se):
> 
> > Do we want to keep [.ci/.ci/.env.test](https://github.com/minvws/nl-kat-coordination/blob/main/.ci/.env.test)? It seems to be empty.

If there are no references to it, then I suppose it can be removed

**Comment by originalsouth** [comment]
Also added changes for the recipes of `make -C octopoes itest` and `make -C octopoes rtest`

**Comment by originalsouth** [comment]
> If there are no references to it, then I suppose it can be removed

Decided to keep it for possible future variables we have to include

**Review by ammar92 — APPROVED** [review]
Looks good to me 👍 If we somehow find a way later to avoid specifying `--project-directory`, that would be my preference

File: bytes/Makefile [hunk 1]
```diff
@@ -60,7 +60,7 @@ migrate: ## Run alembic migrations
 ##|------------------------------------------------------------------------|
 
 test: utest itest ## Run all tests.
-ci-docker-compose := docker compose -f base.yml  -f .ci/docker-compose.yml
```
Comment (Donnype) [inline bytes/Makefile:63]: By changing the paths in the compose file you can sometimes resolve this already even without the --project-directory 👍

Comment (originalsouth) [reply bytes/Makefile:63]: I am not sure what you mean @Donnype, can you be more specific?
* You want to change the path inside the compose file (tried that with `context` wasn't trivial to get to work...)?
* You want to change the path of the compose file?
```
--project-directory string   Specify an alternate working directory
                             (default: the path of the, first specified, Compose file)
```

**Review by dekkers — APPROVED** [review]

**Pull request merged**